elements/bulbs-video/elements/summary.js

```diff
@@ -24,6 +24,19 @@ export function VideoSummaryView (props) {
     );
   }
 
+  let campaign;
+  if (video.tunic_campaign_url) {
+    campaign = (
+      <campaign-display
+          data-track-action={props.campaignTrackAction}   // eslint-disable-line react/prop-types
```

Comment:
Turned out also didn't need this change. ESlint wanted these to be defined in `propTypes`, but there are two `propTypes` in this file which I didn't notice, so I thought this was complaining because these were undefined but that was incorrect.